branching.md

```
@@ -1 +1,3 @@
 Branching strategies for Git
+
+Use [this] (https://nvie.com/posts/a-successful-git-branching-model/) for branching 
```

Markdownlint

Bare URL used

(MD034, no-bare-urls)